projects/cube_xr/CubeXr.cpp

```diff
@@ -32,14 +32,18 @@ void CubeXrApp::Config(ppx::ApplicationSettings& settings)
     settings.grfx.pacedFrameRate        = 0;
     settings.xr.enable                  = true;
     settings.xr.enableDebugCapture      = false;
+    settings.xr.enableMultiView         = true;
```

Comment:
Ideally we set this from a flag instead

Reply:
Not quite, perhaps you can add a KnobFlag<bool> here https://github.com/google/bigwheels/blob/main/include/ppx/application.h#L91 and initialize it here https://github.com/google/bigwheels/blob/main/src/ppx/application.cpp#L670